subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -381,3 +381,15 @@ void wifi_mgmt_raise_raw_scan_result_event(struct net_if *iface,
 					sizeof(*raw_scan_result));
 }
 #endif /* CONFIG_WIFI_MGMT_RAW_SCAN_RESULTS */
+
+void wifi_mgmt_raise_disconnect_complete_event(struct net_if *iface,
+					       int status)
+{
+	struct wifi_status cnx_status = {
+		.status = status,
+	};
+
+	net_mgmt_event_notify_with_info(NET_EVENT_WIFI_CMD_DISCONNECT_COMPLETE,
```

Comment:
Sorry for post merge review, but I verified it doesn't work. I can be fixed by replacing `NET_EVENT_WIFI_CMD_DISCONNECT_COMPLETE` with the `NET_EVENT_WIFI_DISCONNECT_COMPLETE`.